models/RdfImporter.php

```diff
@@ -44,6 +104,116 @@ function ($resource)
         return $report;
     }
 
+    protected function flatImport($content, core_kernel_classes_Class $class)
+    {
+        $report = new Report(Report::TYPE_INFO, __('Data import started'));
```

Comment:
There is some time I don't work with TAO code so I don't know about how hard or the effort needed, but can't we have some kind of `UserImportReportFactory` class and inject as dependency?

Reply:
Not really needed. But good catch, maybe somewhere in future the "report" system itself would be re-written. But now, unfortunately, it's out of scope of current one